conf.py

```diff
@@ -1,10 +1,112 @@
 """
 Configuration for building the website with Sphinx.
 """
-import sys
-import os
+import re
 import datetime
 import subprocess
+from requests import get
+from pathlib import Path
+
+
+AUTHORS_BASE_URL = "https://raw.githubusercontent.com/fatiando/{}/{}/AUTHORS.md"
+AUTHOR_HTML_CARD = """
+<div class="col-6 col-md-3 col-sm-4">
```

Comment:
```suggestion
<div class="col-4 col-sm-3 col-md-2 gx-2">
```

Reduce the column size so we can fit more cards per line. This still looks good but doesn't take up too much vertical space.